com.unity.shadergraph/Editor/Data/Nodes/UV/ParallaxMappingNode.cs

```diff
@@ -70,7 +70,7 @@ public void GenerateNodeCode(ShaderStringBuilder sb, GenerationMode generationMo
             var amplitude = GetSlotValue(kAmplitudeSlotId, generationMode);
             var uvs = GetSlotValue(kUVsSlotId, generationMode);
 
-            sb.AppendLines(String.Format(@"$precision2 {5} = {4} + ParallaxMapping(TEXTURE2D_ARGS({0}.tex, {1}.samplerstate), IN.{2}, {3} * 0.01, {4});",
+            sb.AppendLines(String.Format(@"$precision2 {5} = SHADERGRAPH_TRANSFORM_TEX({4}, {0}) + ParallaxMapping(TEXTURE2D_ARGS({0}.tex, {1}.samplerstate), IN.{2}, {3} * 0.01, SHADERGRAPH_TRANSFORM_TEX({4}, {0}));",
```

Comment:
hmmm.... this might be a little bit of  a weird case... will have to think what should happen here.

Reply:
What makes this case weird?

Reply:
weirdness fixed